pages/common/picom-trans.md

```diff
@@ -0,0 +1,20 @@
+# picom-trans
+
+> Set the window opacity for the `picom` window compositor.
+> More information: <https://github.com/yshui/picom>.
+
+- Set the currently focused window opacity to a specific percentage:
+
+`picom-trans --current --opacity={{90}}`
+
+- Set the opacity of a window with a specific name:
+
+`picom-trans --name={{Firefox}} --opacity={{90}}`
```

Comment:
Just one last thing from my side: Could you please test if those flags work without the `=` and remove them if they do?

Reply:
Sure. They do work, so [I removed them as you requested](https://github.com/tldr-pages/tldr/pull/8244/commits/04d516ca26a97fd7bd6bd158dc1dbec86ad1a625). 

The man page however suggest using `=`. So just out of curiosity: Shouldn't we stick to that?

Reply:
Because it makes it more readable. And it's easier to just never use them instead of remembering where they work and where not. In the end it doesn't make a difference.